pvlib/temperature.py

```diff
@@ -638,6 +638,9 @@ def ross(poa_global, temp_air, noct):
     noct : numeric
         Nominal operating cell temperature [C], determined at conditions of
         800 W/m^2 irradiance, 20 C ambient air temperature and 1 m/s wind.
```

Comment:
It's good to keep PR scope focused, but our units convention was only recently introduced and we aim to update the documentation gradually. Since you're working on this function anyway, I don't think anyone would object to (personally, I would encourage) updating the formatting of these units here and elsewhere in this function. That said, anyone, feel free to object

```suggestion
    noct : numeric
        Nominal operating cell temperature [°C], determined at conditions of
        800 Wm⁻² irradiance, 20 °C ambient air temperature and 1 ms⁻¹ wind speed.
```

Reply:
Thanks for the initiative @RDaxini. I thought about that but then I noticed that the issue applies to the whole file. So to make the PR more focused, I kept the original formatting and moved that to a separate issue (#2519).

If that's okay for you, I would leave this PR with the `Adds` » `Add` and correct the doi formatting (which I've already commited).

Next week I will address the unit styling in #2519, feel free to join the discussion. There I'm proposing to have a `.` between units (e.g. `W.m⁻²`) to make reading easier, and to present the Ross equation with `800` in the denominator to shift from the historically cell-motivated mW.cm⁻² to a more module-friendly W.m⁻² (which also dismisses the current `0.1` factor to match the irradiance unit).

Reply:
I agree, let's do edits for style in #2519 for all of temperature.py.

Reply:
If there is already work planned to do this separately then that's fine